<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawGraphics</class>
 <widget class="QDialog" name="DrawGraphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#121212; color:#ffffff;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCustomPlot" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color:#ffffff</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QLabel{
font-size: 20px;
color:#ffffff;
}</string>
     </property>
     <property name="text">
      <string>X min</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="x_min">
     <property name="styleSheet">
      <string notr="true">background-color:#bb86fc</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">QLabel{
font-size: 20px;
color:#ffffff;
}</string>
     </property>
     <property name="text">
      <string>X max</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="x_max">
     <property name="styleSheet">
      <string notr="true">background-color:#bb86fc</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">QLabel{
font-size: 20px;
color:#ffffff;
}</string>
     </property>
     <property name="text">
      <string>Y min</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="y_min">
     <property name="styleSheet">
      <string notr="true">background-color:#bb86fc</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">QLabel{
font-size: 20px;
color:#ffffff;
}</string>
     </property>
     <property name="text">
      <string>Y max</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="y_max">
     <property name="styleSheet">
      <string notr="true">background-color:#bb86fc</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="draw">
     <property name="styleSheet">
      <string notr="true">QPushButton {
  font-size: 24px;
  text-align: center;
  color: #ffffff;
  background-color: #332940;
  border-radius: 15px;
  border: 5px solid #000000;
}

QPushButton:pressed {
  background-color: #1f1a24;
}</string>
     </property>
     <property name="text">
      <string>Нарисовать график</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
